exercises/matthew_bowles/ex01_basics/inc/Io.h

```diff
@@ -0,0 +1,22 @@
+#ifndef IO_H
+#define IO_H
+
+#include <string>
+#include <map>
+#include <vector>
+
+using namespace std;
```

Comment:
No need to change it now, but note that in Mantid we avoid `using namespace std`, especially in header files. See [here](http://llvm.org/docs/CodingStandards.html#do-not-use-using-namespace-std). If this makes the code too ugly/unreadable, you can use typedefs for commonly-occuring types, e.g.
```
typedef std::map<std::string, int> WordCounts
typedef std::vector<std::pair<std::string, int>> SortedWordCounts
...
//sorts map of words into descending order by value
SortedWordCounts sortWords(const WordCounts& words);
```